### PR TITLE
feat: add Node.js mysql2/mysql.js/Prisma chinese tutorial

### PR DESCRIPTION
**Pull request by Mini256**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs/pull/14607
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by shczhen — APPROVED** [review]

**Comment by sykp241095** [comment]
Why don't start from English version?

File: develop/dev-guide-sample-application-nodejs-mysql2.md [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+summary: 本文描述了 TiDB 和 node-mysql2 的连接步骤，并给出了简单示例代码片段。
+---
+
+# 如何用 node-mysql2 连接到 TiDB
+
+TiDB 是一个兼容 MySQL 的数据库。[node-mysql2](https://github.com/sidorares/node-mysql2) 是一个与 [mysqljs/mysql](https://github.com/mysqljs/mysql) 兼容的面向 Node.js 的 MySQL 驱动。
+
+本文档将展示如何使用 TiDB 和 node-mysql2 来构造一个简单的 CRUD 应用程序。
+
+- 配置你的环境。
```
Comment (sykp241095) [inline develop/dev-guide-sample-application-nodejs-mysql2.md:12]:
```suggestion
- 配置环境。
```

File: develop/dev-guide-sample-application-nodejs-mysql2.md [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+
+# 如何用 node-mysql2 连接到 TiDB
+
+TiDB 是一个兼容 MySQL 的数据库。[node-mysql2](https://github.com/sidorares/node-mysql2) 是一个与 [mysqljs/mysql](https://github.com/mysqljs/mysql) 兼容的面向 Node.js 的 MySQL 驱动。
+
+本文档将展示如何使用 TiDB 和 node-mysql2 来构造一个简单的 CRUD 应用程序。
+
+- 配置环境。
+- 使用 node-mysql2 驱动连接到 TiDB 集群。
+- 构建并运行你的应用程序。你也可以参考[示例代码片段](#示例代码片段)，完成基本的 CRUD 操作。
```
Comment (sykp241095) [inline develop/dev-guide-sample-application-nodejs-mysql2.md:14]:
```suggestion
- 构建并运行应用程序。你也可以参考[示例代码片段](#示例代码片段)，完成基本的 CRUD 操作。
```

Comment (Mini256) [reply develop/dev-guide-sample-application-nodejs-mysql2.md:14]: 没有“你的”就感觉有点奇怪。

通用模板的部分的内容在这个 PR 上 comment 一下吧：https://github.com/pingcap/docs-cn/pull/14902/files

现在模板上达成统一。

File: develop/dev-guide-sample-application-nodejs-mysql2.md [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+summary: 本文描述了 TiDB 和 node-mysql2 的连接步骤，并给出了简单示例代码片段。
+---
+
+# 如何用 node-mysql2 连接到 TiDB
+
+TiDB 是一个兼容 MySQL 的数据库。[node-mysql2](https://github.com/sidorares/node-mysql2) 是一个与 [mysqljs/mysql](https://github.com/mysqljs/mysql) 兼容的面向 Node.js 的 MySQL 驱动。
+
+本文档将展示如何使用 TiDB 和 node-mysql2 来构造一个简单的 CRUD 应用程序。
+
+- 配置环境。
```
Comment (Mini256) [inline develop/dev-guide-sample-application-nodejs-mysql2.md:12]:
```suggestion
- 配置你的环境。
```

File: develop/dev-guide-sample-application-nodejs-prisma.md [hunk 4]
```diff
@@ -0,0 +1,360 @@
+---
+title: 使用 Prisma 连接到 TiDB
+summary: 本文描述了 TiDB 和 Prisma 的连接步骤，并给出了简单示例代码片段。
+---
+
+# 如何用 Prisma 连接到 TiDB
```
Comment (ran-huang) [inline develop/dev-guide-sample-application-nodejs-prisma.md:6]:
```suggestion
# 使用用 Prisma 连接到 TiDB
```

**Review by ran-huang — APPROVED** [review]

**Review by ran-huang — APPROVED** [review]

File: develop/dev-guide-sample-application-nodejs-prisma.md [hunk 5]
```diff
@@ -0,0 +1,360 @@
...
+
+```
+Your database is now in sync with your schema.
+
+✔ Generated Prisma Client (5.1.1 | library) to ./node_modules/@prisma/client in 54ms
+```
+
+这个命令同时会根据 `prisma/schema.prisma` 文件中的模型定义，生成用于与数据库交互的 [Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client) 的代码。
+
+
```
Comment (ran-huang) [inline develop/dev-guide-sample-application-nodejs-prisma.md:227]:
```suggestion
```

**Review by sykp241095 — APPROVED** [review]

**Pull request merged**